src/ByteDev.Sonos.Upnp/Services/AvTransportService.cs

```diff
@@ -168,5 +168,10 @@ public async Task<GetTransportSettingsResponse> GetTransportSettingsAsync()
 
             return new GetTransportSettingsResponseFactory().CreateFor(ActionXNamespace, xml);
         }
+
+        public async Task SetPlayModeAsync()
```

Comment:
@ifcodesdotnet Hi, what's the point of this method? It isn't called and doesn't supply a PlayMode type like the existing overload above. Was accidental commit? :-) Thanks

Reply:
I've committed a change to remove the unnecessary method! Sorry !